KHLY2700/KHLY2700-IOC-01App/Db/keithley2700.db

```diff
@@ -0,0 +1,733 @@
+}
+
+record(ai, "$(P)CH:START")
+{
+	field(DESC, "Return first channel in scan range")
+	field(DTYP, "stream")
+	field(SCAN, "1 second")
+	field(INP, "@devKeithley_2700.proto getChannelsStart $(PORT)")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:CH:START")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ai, "$(P)CH:END")
+{
+	field(DESC, "Return last channel in scan range")
+	field(DTYP, "stream")
+	field(SCAN, "1 second")
+	field(INP, "@devKeithley_2700.proto getChannelsEnd $(PORT)")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:CH:END")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)CH:END:SP")
+{
+	field(DESC, "Set end point of channel range to scan")
+	field(DTYP, "stream")
+	field(OUT, "@devKeithley_2700.proto setScanChannelsEnd($(P)CH) $(PORT)")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:CH:END:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(bo, "$(P)BUFFER:CLEAR:SP")
+{
+	field(DESC, "Clear readings from buffer")
+	field(DTYP, "stream")
+	field(ZNAM, "")
+	field(ONAM, "")
+	field(OUT, "@devKeithley_2700.proto clearBuffer $(PORT)")    
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER:CLEAR:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)BUFFER:CLEAR:SP","$(P)BUFFER:CLEAR")
+
+record(bo, "$(P)SYSTEM:CLEAR:SP")
+{
+	field(DESC, "Clear error message from queue")
+	field(DTYP, "stream")
+	field(ZNAM, "")
+	field(ONAM, "")
+	field(OUT, "@devKeithley_2700.proto clearSystemErrors $(PORT)")  
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SYSTEM:CLEAR:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)SYSTEM:CLEAR:SP","$(P)SYSTEM:CLEAR")
+
+record(mbbo, "$(P)MEASUREMENT:SP") {
+	field(DESC, "Set measurement function")
+	field(DTYP, "stream")
+	field(ZRST, "VOLT")
+    field(ONST, "VOLT:AC")
+    field(TWST, "CURR")
+    field(THST, "CURR:AC")
+    field(FRST, "RES")
+    field(FVST, "FRES")
+    field(SXST, "CONT")
+    field(SVST, "FREQ")
+    field(EIST, "PER")
+	field(OUT, "@devKeithley_2700.proto setMeasurement($(P)CH) $(PORT)")  
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:MEASUREMENT:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)MEASUREMENT:SP","$(P)MEASUREMENT")
+
+record(mbbo, "$(P)BUFFER:SOURCE:SP") 
+{
+    field(DESC, "Get buffer feed source")
+    field(SCAN, "1 second")
+    field(DTYP, "stream")
+    field(OUT,  "@devKeithley_2700.proto setBufferFeedSource $(PORT)")
+    field(ZRST, "SENS")
+    field(ONST, "CALC")
+    field(TWST, "NONE")
+	info(ARCHIVE, "VAL")
+    info(INTEREST, "HIGH")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER:SOURCE:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)BUFFER:SOURCE:SP", "$(P)BUFFER:SOURCE")
+
+record(bo, "$(P)BUFFER:CLEAR:AUTO:SP") 
+{
+    field(DESC, "Set Buffer State")
+    field(SCAN, "1 second")
+    field(DTYP, "stream")
+    field(OUT,  "@devKeithley_2700.proto setBufferAutoClearStatus $(PORT)")
+    field(ZNAM, "ON")
+    field(ONAM, "OFF")
+	info(ARCHIVE, "VAL")
+    info(INTEREST, "HIGH")
```

Comment:
Is this really high interest? My basic measure is "Would I have a block that points to this?". I don't think users would for the auto clear status of the buffer. Please could you have a look elsewhere where you've used high interest and check they're appropriate. There is a "MEDIUM" interest too if you really want these to be interesting.